frontend/src/employee-frontend/components/vasu/templates/VasuTemplateEditor.tsx

```diff
@@ -423,6 +435,7 @@ export default React.memo(function VasuTemplateEditor() {
                               }
                               text={i18n.vasuTemplates.addNewQuestion}
                               icon={faPlus}
+                              disabled={readonly}
```

Comment:
Would it make more sense to drop these buttons completely if the template is readonly?

Reply:
https://github.com/espoon-voltti/evaka/blob/60f8035d0c6ec607f6465ad807a8e3bdb577a648/service/src/main/kotlin/fi/espoo/evaka/vasu/VasuTemplateController.kt#L149 backend validation actually is there already. You were probably looking at the other PUT endpoint which updates e.g. template date range and allows limited edits even after template is in use.

Reply:
Sorry, my bad.